static/js/message_list.js

```diff
@@ -673,6 +673,14 @@ $(document).on('message_selected.zulip zuliphashchange.zulip mousewheel', functi
     viewport.stop_auto_scrolling();
 });
 
+$(window).load(function () {
+    $('img.emoji').each(function () {
```

Comment:
We might want to just do this for the few classes of images that users upload (inline images + emoji, or something), so we don't need to think about every imaged used on the site.  Also, probably we want a different error image than a zulip icon; ideas for what's good?

@brockwhittaker what do you think?